testsuite/MDAnalysisTests/topology/test_mol2.py

```diff
@@ -30,13 +30,15 @@
     mol2_molecule,
     mol2_molecules,
 )
-from numpy.testing import assert_equal
 
+import numpy as np
+from io import StringIO
```

Comment:
should these be moved about the MDA imports (I feel like theres a PEP for this but I can never remember), also shouldn't there be two lines between imports and the class (might be forgetting my PEP8)?